Foundation/HTTPCookie.swift

```diff
@@ -444,14 +488,16 @@ open class HTTPCookie : NSObject {
     }
 
     //Bake a cookie
-    private class func createHttpCookie(url: URL, pairs: ArraySlice<String>) -> HTTPCookie? {
-        var properties: [HTTPCookiePropertyKey : Any] = [:]
+    private class func createHttpCookie(url: URL, cookie: Substring) -> HTTPCookie? {
+        let pairs = cookie.components(separatedBy: ";") // split the name/value and attribute/value pairs
+                          .map({$0.trim()})             // trim whitespaces
+                          .filter({$0.caseInsensitiveCompare("HTTPOnly") != .orderedSame})  // we don't use HTTPOnly, do we?
+                          .flatMap(HTTPCookie.createNameValuePair(pair:)) // create Name and Value properties
+
```

Comment:
Nit: this can be
```
let pairs = cookie.components(separatedBy: ";") // split the name/value and attribute/value pairs
	                     .map { $0.trim() }            // trim whitespaces
	                     .filter { $0.caseInsensitiveCompare("HTTPOnly") != .orderedSame }   // we don't use HTTPOnly ...
	                     .flatMap(HTTPCookie.createNameValuePair) // create Name and Value properties
```

Reply:
I will change it. I tried to keep the old code, just in case the diff was simpler, but since it is not, there's no reason to keep the parenthesis.